.gitignore

```diff
@@ -32,3 +32,4 @@ tests/tmp.*
 *.bundle
 *.log
 dqlite-deps.tar.bz2
+dump/
```

Comment:
Only because we've seen in the past if we don't add sane .gitignore we end up vendoring files we shouldn't have (it's not even once this has happened).

Reply:
Removed it, as we default to a temp directory now.